Shopping/src/test/java/com/asos/Resources/register.feature

```diff
@@ -0,0 +1,55 @@
+Feature: Register to the ASOS page
+  As a user
+  I want to register the asos site
+  So that I can register successfully.
+
+  Scenario Outline: user can register an account with valid data.
+    Given user is on Asos Join/registration  page
+    When  user should enter the "<emailaddress>" "<firstname>" "<lastname>" "<password>" "<dateofbirth>""<gender>"
+    And   user clicks join ASOS button
+    Then  user should navigate to home page with logged in state
+
+    Examples:
+      | emailaddress              | firstname |lastname  | password  | dateofbirth | gender |
+      | jancilondon1@gmail.com    |janci      | sarasasi1   | sarasasi1 | 17/04/1984    |female|
+
+
+  Scenario Outline: email address validation
+    Given user is in asospage
+    When  he should enter the invalid emailaddress "<emailaddress>" "<firstname>" "<lastname>" "<password>" "<dateofbirth>""<gender>"
+    And   he clicks  join ASOS button
+    Then  "Email fail! Please type in your correct email address"  message is displayed
+    Examples:
+      | emailaddress              | firstname |lastname  | password  | dateofbirth | gender |
+      | 365676787899890            |yyuuiu    | hgfhghj  |5768798 | hghjjkh    |77778|
+
+  Scenario Outline: password field validation.
+    Given user is in asospage
```

Comment:
follow same pre-condition as you did for first scenario
Given user is on Asos Join/registration  page